labels/ethereum/balancer_arbitrage.sql

```diff
@@ -0,0 +1,29 @@
+SELECT
+    DISTINCT(t.to) AS address,
+    'abritrage_bot' AS label,
+    'dapp usage' AS type,
+    'balancerlabs' AS author
+FROM dex.trades t1
+INNER JOIN dex.trades t2
+ON t1.tx_hash = t2.tx_hash
+AND t1.token_a_address = t2.token_b_address 
+AND t1.token_b_address = t2.token_a_address 
+AND ((t1.project = 'Balancer' AND t2.project = 'Uniswap') or (t1.project = 'Uniswap' AND t2.project = 'Balancer'))
+INNER JOIN ethereum.transactions t ON t.hash = t1.tx_hash
+WHERE date_trunc('day', t1.block_time) >= date_trunc('day', now() - interval '2 months')
+AND date_trunc('day', t2.block_time) >= date_trunc('day', now() - interval '2 months')
```

Comment:
Could make sense to use the `{{timestamp}}` described here? https://github.com/duneanalytics/abstractions/tree/master/labels

Reply:
I believe it's a good solution, but if we substitute the `WHERE` clauses for it, would it be able to run it for the first time since it would search the whole `dex.trades`?

Reply:
Yes, it should be OK!